models/vae_parameters.json

```diff
@@ -0,0 +1,74 @@
+{
+	"paths":
+		{
+			"simulated_2d": "cryo_sim_128x128.npy",
+			"real_2d30": "class2D_30_sort.h5",
+			"real_2d39": "class2D_39_sort.h5",
+			"real_2d93": "class2D_93_sort.h5",
+			"refine_3D": "refine3D_180x180_sort.h5",
+			"class_3d9090": "class3D_90x90_sort.h5",
+			"simulated_3d": "concat_simulated.npy",
+			"simulated_3d_noise": "cryo_sim_128x128.npy",
+			"real_3d": "data.hdf5",
+			"4points": "4points.npy",
+			"4points1": "4points1.npy",
```

Comment:
what's the difference between 4points and 4points1? maybe use more descriptive names?